code/modules/atmospherics/environmental/LINDA_fire.dm

```diff
@@ -240,6 +240,15 @@
 		var/mob/living/L = AM
 		L.fire_act(temperature, volume)
 
+/obj/effect/hotspot/backblast
+	volume = 250
+
+/obj/effect/hotspot/backblast/Crossed(atom/movable/AM, oldLoc)
+	..()
+	if(isliving(AM))
+		var/mob/living/L = AM
+		L.adjust_fire_stacks(5) // +5 firestacks from normal
+
```

Comment:
and this is just to make it more spooky I assume?

Reply:
yeah, and i lowered the number to +3